### PR TITLE
Fix icon layout in action buttons

### PR DESCRIPTION
**Pull request by GulomovCreative**
### What does it do?
Fix icon layout in action buttons.

### Why is it needed?
Icons have different widths and because of this they are not aligned.

**Before:**

screenshot-revolution-2019 03 04-01-41-35

**After:**

screenshot-revolution-2019 03 04-01-39-59

### Related issue(s)/PR(s)
None

### CODE REVIEW

**Review by Mark-H — DISMISSED** [review]

**Review by Jako — DISMISSED** [review]

File: _build/templates/default/sass/index.scss [hunk 1]
```diff
@@ -1456,6 +1456,7 @@ iframe[classname="x-hidden"] {
       .icon {
         text-align: center;
         margin-left: 10px;
+        width: 18px;
```
Comment (JoshuaLuckers) [inline _build/templates/default/sass/index.scss]:
```suggestion
        @extend .#{$fa-css-prefix}-fw;
```

To prevent unnecessary changes down the road It would be better to use the fixed-width utility FontAwesome provides. If my suggestion is approved you can also remove the `text-align: center;`.

**Review by JoshuaLuckers — CHANGES_REQUESTED** [review]

File: _build/templates/default/sass/index.scss (hunk 1 above)
Comment (Jako) [reply _build/templates/default/sass/index.scss]: Thats better!

**Review by JoshuaLuckers — APPROVED** [review]

**Comment by JoshuaLuckers** [comment]
Thanks @GulomovCreative 👍

**Comment by GulomovCreative** [comment]
> Thanks @GulomovCreative 👍

Not at all, thank you @JoshuaLuckers

**Review by Jako — APPROVED** [review]

**Review by Mark-H — APPROVED** [review]

**Comment by Mark-H** [comment]
Merged, thanks.
